### PR TITLE
Add release process to README

### PR DESCRIPTION
**Pull request by will-moore**
From https://github.com/ome/omero-gallery/pull/68#issuecomment-675059296

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -207,6 +207,17 @@ like: ``idr0001-graml-sysgro/screenA``::
...
+Release process
+---------------
+
+- review and update the CHANGELOG
+- run `bumpversion release` to remove the dev suffix and create a signed tag
+- run `bumpversion --no-tag patch` to bump the version to the next dev suffix
+- push the newly created tag and `master` to `origin`. e.g. `git push origin master v3.3.3`
+- the Travis CI build for the tag will also include a PyPI deployment step
+
+https://github.com/ome/omero-gallery/blob/352a9bcc227bf35ca63a3735b19d1a641dab0af5/.travis.yml#L17-L23
```
Comment (joshmoore) [inline README.rst:219]: I probably wouldn't include this link in the README.

File: README.rst [hunk 2]
```diff
@@ -207,6 +207,15 @@ like: ``idr0001-graml-sysgro/screenA``::
         {"key":"Name", "regex": "^(.*?)-.*?(.)$", "template": "$1$2"},
     ]'
 
+Release process
+---------------
+
+- review and update the CHANGELOG
```
Comment (pwalczysko) [inline README.rst:213]:
```suggestion
- review and update the [CHANGELOG](https://github.com/ome/omero-gallery/blob/master/CHANGELOG.md)
```

File: README.rst [hunk 3]
```diff
@@ -207,6 +207,15 @@ like: ``idr0001-graml-sysgro/screenA``::
...
+- run `bumpversion release` to remove the dev suffix and create a signed tag
```
Comment (pwalczysko) [inline README.rst:214]:
```suggestion
- run ``bumpversion release`` to remove the dev suffix and create a signed tag
```

File: README.rst [hunk 4]
```diff
@@ -207,6 +207,15 @@ like: ``idr0001-graml-sysgro/screenA``::
...
+- run `bumpversion --no-tag patch` to bump the version to the next dev suffix
```
Comment (pwalczysko) [inline README.rst:215]:
```suggestion
- run ``bumpversion --no-tag patch`` to bump the version to the next dev suffix
```

This one and the previous above did not render nicely, so trying to add 2 backticks.

File: README.rst [hunk 5]
```diff
@@ -207,6 +207,15 @@ like: ``idr0001-graml-sysgro/screenA``::
...
+- push the newly created tag and `master` to `origin`. e.g. `git push origin master v3.3.3`
```
Comment (pwalczysko) [inline README.rst:216]:
```suggestion
- push the newly created tag and ``master`` to ``origin``. e.g. ``git push origin master v3.3.3``
```

File: README.rst [hunk 6]
```diff
@@ -207,6 +207,15 @@ like: ``idr0001-graml-sysgro/screenA``::
...
+- the Travis CI build for the tag will also include a PyPI deployment step
```
Comment (pwalczysko) [inline README.rst:217]:
```suggestion
- the Travis CI build for the tag includes a PyPI deployment step, so no need to deploy on PyPi manually
```

**Comment by pwalczysko** [comment]
Couple of suggestions, otherwise makes sense and reads well, but of course I did not test the steps.

**Comment by pwalczysko** [comment]
Ready to merge FMPOV

**Comment by joshmoore** [comment]
:tada:

**Pull request merged**